scripts/require-sync/main.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/open-policy-agent/frameworks/constraint/pkg/core/templates"
 	gkapis "github.com/open-policy-agent/gatekeeper/v3/apis"
 	"github.com/open-policy-agent/gatekeeper/v3/pkg/gator/reader"
-	"github.com/open-policy-agent/gatekeeper/v3/pkg/syncutil/parser"
+	"github.com/open-policy-agent/gatekeeper/v3/pkg/cachemanager/parser"
```

Comment:
Looks like the change in path name results in a different import ordering.  You can fix automatically with `golangci-lint run main.go --fix` or `gofumpt -w main.go`.